tests/unit/combine_results/combined_MotEvo_results.tsv

```diff
@@ -0,0 +1,22 @@
+strand	binding_position	binding_posterior	pwm_id	binding_region	binding_sequence	binding_energy	fasta_record
++	14-18	0.015291	HNRNPF_820	"MOTEVO_input-sequence
+"	GTGGG	2.51049	"MOTEVO_input-sequence
+"
++	6-10	0.017773	HNRNPF_823	"MOTEVO_input-sequence
+"	AGTGA	2.51049	"MOTEVO_input-sequence
+"
++	15-19	0.018685	HNRNPF_823	"MOTEVO_input-sequence
+"	TGGGA	2.51049	"MOTEVO_input-sequence
+"
++	46-50	0.674978	HNRNPF_823	"MOTEVO_input-sequence
+"	AGGGA	6.71171	"MOTEVO_input-sequence
+"
++	15-19	0.560144	HNRNPF_824	"MOTEVO_input-sequence
+"	TGGGA	6.71171	"MOTEVO_input-sequence
+"
++	46-50	0.030141	HNRNPF_824	"MOTEVO_input-sequence
+"	AGGGA	2.51049	"MOTEVO_input-sequence
+"
++	51-55	0.014195	HNRNPF_824	"MOTEVO_input-sequence
+"	TCGGA	2.51049	"MOTEVO_input-sequence
```

Comment:
I am very confused by the information here... why is every record separated into two rows? Each record should be one line (i.e. one row) in the result table. The records should match the header of the table.

Reply:
That is because you are viewing the tsv file in GitHub. Have you checked it using LibreOffice?

Reply:
Oh, OK, thanks! GitHub confused me...